sourcetools/j9constantpool/com/ibm/oti/VMCPTool/ConstantPoolStream.java

```diff
@@ -329,7 +329,7 @@ private void writeHeader(int size) {
 			out.println("} _jclROMClass = {");
 
 			String shapeDescriptionOffset = "sizeof(J9ROMClass) - offsetof(J9ROMClass, cpShapeDescription) + " + size;
-			out.println("\t{0,0,0,0,0,J9AccClassUnsafe, 0,0,0,0,0,0,0,0, " + itemCount + ", " + itemCount + ", 0,0,0, " + shapeDescriptionOffset + ", 0,},");
+			out.println("\t{0,0,0,0,0,0, 0,0,0,0,0,0,0,0, " + itemCount + ", " + itemCount + ", 0,0,0, " + shapeDescriptionOffset + ", 0,},");
```

Comment:
This change is unnecessary now (since we're keeping unsafe).

Reply:
Reverted.